src/components/Tree/DataZTree/components/ZTree/index.vue

```diff
@@ -376,7 +376,6 @@ export default {
   .ztree {
     width: 100%;
     overflow: auto;
-    z-index: 1000;
     height: 648px;
     background-color: #ffffff;
 
```

Comment:
The only difference between the two codes is the addition of a new line `height: 648px;` which does not affect performance nor functionality. There is no need to make this change.

Therefore,
- No issues/irregularities found.
- Performance/Centric changes (if there are any) have been made but they can be ignored for the current context.